packages/preact/src/index.ts

```diff
@@ -128,7 +128,7 @@ function Text(this: ComponentType, { data }: { data: Signal }) {
 
 		// Replace this component's vdom updater with a direct text one:
 		currentUpdater!._updater = () => {
-			(this.base as Text).data = s._value;
+			(this.base as Text).data = currentSignal.value.value;
```

Comment:
This causes `true` to render as the text "true", instead of using the logic on L137 to make it match VDOM semantics.

Reply:
Looks like we're missing a test case there